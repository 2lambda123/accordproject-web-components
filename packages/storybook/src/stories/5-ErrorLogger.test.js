import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({
  storyKindRegex: /Error Logger/,
  suite: 'Error Logger Storyshots'
});
