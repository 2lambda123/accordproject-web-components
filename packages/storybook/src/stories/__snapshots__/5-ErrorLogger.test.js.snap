// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Logger Storyshots Error Logger Error Logger 1`] = `
<div
  className="ui-components__error-wrapper"
>
  <div
    className="sc-hBMVcZ hPvWKK ui-components__error-header"
    onClick={[Function]}
  >
    <i
      aria-hidden="true"
      className="exclamation triangle small icon sc-fnVYJo bBsJEB"
      onClick={[Function]}
    />
    1
     
    Error
    <div
      className="sc-fFSRdu bgXqtm ui-components__error-bar-arrow"
    />
  </div>
</div>
`;
